tests/php/test-class-amp-auto-lightbox-disable-sanitizer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Class AMP_Auto_Lightbox_Disable_Sanitizer_Test.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+use AmpProject\AmpWP\Tests\Helpers\MarkupComparison;
+use AmpProject\AmpWP\Tests\TestCase;
+
+/**
+ * Tests the auto lightbox disable sanitizer class.
+ *
+ * @coversDefaultClass AMP_Auto_Lightbox_Disable_Sanitizer
+ */
+class AMP_Auto_Lightbox_Disable_Sanitizer_Test extends TestCase {
+
+	use MarkupComparison;
+
+	/**
+	 * @covers ::sanitize()
+	 */
+	public function test_sanitize() {
```

Comment:
Should we also test if the `amp_auto_lightbox_disabled` filter does its job well?

Reply:
I have added test cases for it in `tests/php/test-amp-helper-functions.php`